src/odemis/gui/comp/overlay/base.py

```diff
@@ -340,12 +339,12 @@ def __init__(self, cnvs, label=None):
 
     def activate(self):
         """ Process user generated mouse events """
-        self.active = True
+        self.active.value = True
         self.cnvs.Refresh()
 
     def deactivate(self):
         """ Stop processing user generated mouse events """
-        self.active = False
+        self.active.value = False
         self.cnvs.Refresh()
```

Comment:
With a VA, typically these functions shouldn't be public. Instead the VA should be written, and a subscriber would call the `.Refresh()`. 
The only drawback is that it means changing every code which calls any of these two functions... which is quite a bit. I'd rather do it now than have this weird interface left over. And, with a clever search and replace, it should be fast!